src/SonarScanner.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -482,7 +482,7 @@
   <Target Name="SetRazorCodeAnalysisProperties" BeforeTargets="RazorCoreCompile" AfterTargets="CopyFilesToOutputDirectory">
     <PropertyGroup Condition=" $(ErrorLog) == $(SonarCompileErrorLog) ">
       <!-- Keep the path so that we could set SonarQubeSetting with sonar.language.roslyn.reportFilePaths -->
-      <RazorSonarCompileErrorLog>$(TargetDir)$(TargetName)$(RazorTargetNameSuffix)$(TargetExt).RoslynCA.json</RazorSonarCompileErrorLog>
+      <RazorSonarCompileErrorLog>$(RazorSonarCompileErrorLog)</RazorSonarCompileErrorLog>
```

Comment:
Isn't this a copy/paste issue? It's setting Sonar version `RazorSonarCompileErrorLog` to the same Sonar version `RazorSonarCompileErrorLog`.

I think we don't need `RazorSonarCompileErrorLog` anymore. So I'd do this:
```
<RazorCompilationErrorLog>$(TargetDir)$(TargetName)$(RazorTargetNameSuffix)$(TargetExt).RoslynCA.json</RazorCompilationErrorLog>
<ErrorLog>$(RazorCompilationErrorLog)</ErrorLog> 
```